.github/workflows/build.yml

```diff
@@ -166,10 +174,41 @@ jobs:
           context: ./packages/indexer
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:indexer
+
+  drop_db:
+    runs-on: ubuntu-latest
+    needs: [build_api, build_indexer, test_data_contract, test_api, test_frontend]
+    
+    if: (github.event_name != 'pull_request' || github.event.pull_request.merged == true)
+    
+    steps:
+      - uses: Stockopedia/action-get-changed-files@v1
```

Comment:
We don't need to check changed state. Simply ignore if it is not tagged push

Reply:
✅